main/res/values/strings.xml

```diff
@@ -791,6 +791,8 @@
     <string name="init_summary_bigSmileysOnMap">If enabled big emoticons will be shown for logged caches on the map instead of the cache type icon (requires restart).</string>
     <string name="init_global_wp_extraction_disable">Disable waypoint extraction</string>
     <string name="init_summary_global_wp_extraction_disable">Globally disable waypoint extraction from personal cache notes. It can still be enabled for individual caches by deactivating the checkbox below the personal cache note</string>
+    <string name="init_prevent_cache_note_merge">Prevent merging of Personal Cache Notes</string>
+    <string name="init_summary_prevent_cache_note_merge">If this setting is enabled c:geo will delete the locally stored note permanently and replace it by the personal note on the server. No merging will be done</string>
```

Comment:
We should use "note" instead of "notes" (singular form) in all texts, imho.
E.g., iIt is used in this way at the geocaching.com website.

The "." at the end of the last sentences are missing (for init_summary_global_wp_extraction_disable and init_summary_prevent_cache_note_merge).

Reply:
ok, done. Both strings you mentioned now have terminating dots.